.github/test-matrix.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
i think this file is unused at the moment

Reply:
It's just unused for the moment to get this merged.